drivers/serial/uart_neorv32.c

```diff
@@ -12,6 +12,7 @@
 #include <pm/device.h>
 #include <sys/sys_io.h>
 
+#define LOG_LEVEL CONFIG_UART_LOG_LEVEL
 #include <logging/log.h>
 LOG_MODULE_REGISTER(uart_neorv32, CONFIG_LOG_DEFAULT_LEVEL);
```

Comment:
```suggestion
#include <logging/log.h>
LOG_MODULE_REGISTER(uart_neorv32, CONFIG_UART_LOG_LEVEL);
```

Which log level is supposed to be used here?